interface-definitions/include/dhcp/ddns-settings.xml.i

```diff
@@ -0,0 +1,172 @@
+<!-- include start from dhcp/ddns-settings.xml.i -->
+<leafNode name="send-updates">
+    <properties>
+        <help>Enable or disable updates for this scope</help>
+        <completionHelp>
+            <list>enable disable</list>
+        </completionHelp>
+        <valueHelp>
+            <format>enable</format>
+            <description>Enable updates for this scope</description>
+        </valueHelp>
+        <valueHelp>
+            <format>disable</format>
+            <description>Disable updates for this scope</description>
+        </valueHelp>
+        <constraint>
+            <regex>(enable|disable)</regex>
+        </constraint>
+        <constraintErrorMessage>Set it to either enable or disable</constraintErrorMessage>
+    </properties>
+</leafNode>
+<leafNode name="force-update-both">
```

Comment:
I think the name from Kea, `override-client-update` is clearer than `force-update-both`. I also cannot see a reason not to make it valueless here. All those things are boolean in the parser grammar, do they have a potential to become tri-state or more?